core-feature-pack/pom.xml

```diff
@@ -786,6 +786,16 @@
             <groupId>org.wildfly.core</groupId>
             <artifactId>wildfly-launcher</artifactId>
         </dependency>
+        
+        <dependency>
+            <groupId>org.wildfly.core</groupId>
+            <artifactId>wildfly-jar-runtime</artifactId>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.wildfly.core</groupId>
+            <artifactId>wildfly-jar-boot</artifactId>
+        </dependency>
```

Comment:
We need to add these to the license `core-feature-pack-licenses.xml` with the appropriate license.

Reply:
Thank-you, added.